flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -279,7 +498,7 @@ public void alterTable(CatalogBaseTable table, ObjectIdentifier objectIdentifier
 	 *
 	 * @param objectIdentifier The fully qualified path of the table to drop.
 	 * @param ignoreIfNotExists If false exception will be thrown if the table or database or catalog to be altered
-	 * does not exist.
+	 *                          does not exist.
```

Comment:
minor: revert

Reply:
This was requested by @twalthr to indent long javadocs